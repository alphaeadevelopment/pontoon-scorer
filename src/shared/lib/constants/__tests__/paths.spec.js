import paths from '../paths';

describe('paths', () => {
  it('matches snapshot', () => {
    expect(paths).toMatchSnapshot();
  });
});
