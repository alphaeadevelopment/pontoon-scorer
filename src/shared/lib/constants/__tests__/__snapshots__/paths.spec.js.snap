// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paths matches snapshot 1`] = `undefined`;
